src/.babelrc

```diff
@@ -1,7 +1,7 @@
 {
   "compact": false,
   "presets": [
-    ["es2015", { "loose": true }],
+    ["env", { "loose": true }],
```

Comment:
When you use env without `target` option, it  behaves like `preset-latest` and will enable all transformations that comes in `es2015`, `es2016`, `es2017`. Options should contain `{ target: {node: "current"}}` to enable node feature detection. https://babeljs.io/docs/plugins/preset-env/#usage https://babeljs.io/docs/plugins/preset-latest/